src/routers/secure/scrape.py

```
@@ -337,27 +339,43 @@ def get_media_item(
+            from program.apis.tmdb_api import TMDBConnectionError
+            from program.apis.tvdb_api import TVDBConnectionError
 
-                item = session.merge(indexed)
-                session.commit()
-                session.refresh(item)
-                return item
+            if isinstance(e, TVDBConnectionError):
+                raise HTTPException(
+                    status_code=503,
+                    detail=f"TVDB Service Unavailable: {str(e)}",
+                )
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)